src/UrlParams.ts

```diff
@@ -331,12 +331,15 @@ let urlParamCache: {
 export const getUrlParams = (
   search = window.location.search,
   hash = window.location.hash,
+  /** Skipping the cache might be needed in tests, to allow recomputing based on mocked platform changes. */
```

Comment:
I think this will be needed in every test, for test environment hygiene reasons. Hm, I wonder how we can do this without having to hack around for every test.

Reply:
Exporting a func to reset sounds like a good compromise to me.